app/models/public_body.rb

```diff
@@ -216,6 +216,14 @@ def set_api_key!
     self.api_key = SecureRandom.base64(33)
   end
 
+  def self.find_by_name(name)
+    find_by(name: name)
+  end
+
+  def self.find_by_url_name(url_name)
```

Comment:
`PublicBody.find_by_url_name` only appears in a couple of controllers (and some specs) so this can also be removed relatively easily